english/make-hot-chocolate/hot-choc.md

```diff
@@ -0,0 +1,21 @@
+You have a word with the marshmallow who tried to grab the lighter.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.